src/clr-angular/forms/common/if-error/if-error.service.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Injectable, OnDestroy} from "@angular/core";
+import {NgControl} from "@angular/forms";
+import {Observable, Subject, Subscription} from "rxjs";
+import {filter} from "rxjs/operators";
+
+import {NgControlService} from "../providers/ng-control.service";
+
+@Injectable()
+export class IfErrorService implements OnDestroy {
+    // Implement our own since it doesn't fire on events like blur, and return the control instead of a string
```

Comment:
Didn't understand this comment. `since it doesn't fire on events` do you mean the status isn't updated on events like `blur`